ivy/functional/frontends/paddle/tensor/math.py

```diff
@@ -352,6 +352,23 @@ def frac(x, name=None):
     return ivy.subtract(x, y)
 
 
+@with_supported_dtypes({"2.5.0 and below": ("float32", "float64")}, "paddle")
+@to_ivy_arrays_and_back
+def logcumsumexp(x, axis=None, dtype=None, name=None):
+    if len(x.shape) == 0:
+        ret = x
+    else:
+        # x is input
+        # name is out
+        original_dtype = dtype
+        exp_input = ivy.exp(x.astype("float64"))
+        summed_exp_input = ivy.cumsum(exp_input, axis=axis)
+        ret = ivy.log(summed_exp_input).astype(original_dtype)
+    if ivy.exists(name):
```

Comment:
not the right use of name - https://www.paddlepaddle.org.cn/documentation/docs/en/api/paddle/logsumexp_en.html